src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs

```diff
@@ -66,16 +66,183 @@ public bool GetBooleanValue()
         }
 
         /// <summary>
-        /// Reads the next JSON token value from the source and parses it to a <see cref="int"/>.
+        /// Reads the next JSON token value from the source and parses it to an <see cref="int"/>.
+        /// Returns the value if the entire UTF-8 encoded token value can be successfully parsed to an <see cref="int"/>
+        /// value.
+        /// Throws exceptions otherwise.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
+        /// Thrown if trying to get the value of JSON token that is not a <see cref="JsonTokenType.Number"/>.
```

Comment:
nit: `...value of a JSON token...`

(here and elsewhere)

Reply:
hmmm rather should it be `value of the JSON token` as we have specific token in hand. Line 69 also has same usage (`the` instead of `a`).